End_of_year_Reallocations/credit_test.py

```diff
@@ -42,21 +46,41 @@
         # get excldclosedacct flag
         .assign(excldclosedacct = lambda x: np.where((x['REPCODE'] == 'CL') & (x['lbu_py_tot'] == 0),'Y','N'))
 
+        # get cust level billing
+        .assign(cust_subscr = lambda x: x.groupby('CUSTOMER').adj_lbu_subscr.transform('sum'))
+        .assign(cust_cae_val = lambda x: x.groupby('CUSTOMER').adj_lbu_cae_val.transform('sum'))
+        .assign(cust_cae_p = lambda x: x['cust_cae_val'] / x['cust_subscr'])
 
+        .assign(cus_cy_on_amt = lambda x: x.groupby('CUSTOMER').adj_lbu_cy_on_amt.transform('sum'))
+        .assign(cus_cy_oa_amt = lambda x: x.groupby('CUSTOMER').adj_lbu_cy_oa_amt.transform('sum'))
+        .assign(cus_cy_pr_amt = lambda x: x.groupby('CUSTOMER').adj_lbu_cy_pr_amt.transform('sum'))
+        .assign(cus_cy_tot = lambda x: x.groupby('CUSTOMER').lbu_cy_tot.transform('sum'))
 
-        #.assign(adj_cus_cy_on_amt = lambda x: x['CUS_CY_ON_AMT'].fillna(0) - x['CREDIT_CY_ON_y'].fillna(0))
-        #.assign(adj_cus_cy_oa_amt = lambda x: x['CUS_CY_OA_AMT'].fillna(0) - x['CREDIT_CY_OA_y'].fillna(0))
-        #.assign(adj_cus_cy_pr_amt = lambda x: x['CUS_CY_PR_AMT'].fillna(0) - x['CREDIT_CY_PR_y'].fillna(0))
+        .assign(cus_py_on_amt = lambda x: x.groupby('CUSTOMER').adj_lbu_py_on_amt.transform('sum'))
+        .assign(cus_py_oa_amt = lambda x: x.groupby('CUSTOMER').adj_lbu_py_oa_amt.transform('sum'))
+        .assign(cus_py_pr_amt = lambda x: x.groupby('CUSTOMER').adj_lbu_py_pr_amt.transform('sum'))
+        .assign(cus_py_tot = lambda x: x.groupby('CUSTOMER').lbu_py_tot.transform('sum'))
 
-        #.assign(adj_cus_py_on_amt = lambda x: x['CUS_PY_ON_AMT'].fillna(0) - x['CREDIT_PY_ON_y'].fillna(0))
-        #.assign(adj_cus_py_oa_amt = lambda x: x['CUS_PY_OA_AMT'].fillna(0) - x['CREDIT_PY_OA_y'].fillna(0))
-        #.assign(adj_cus_py_pr_amt = lambda x: x['CUS_PY_PR_AMT'].fillna(0) - x['CREDIT_PY_PR_y'].fillna(0))
-        )
+        .assign(cus_has_cy_online_spend = lambda x: np.where(x['cus_cy_on_amt'] > 0, 'Y', 'N'))
+        .assign(cus_has_py_online_spend = lambda x: np.where(x['cus_py_on_amt'] > 0, 'Y', 'N'))
 
+        # add unique id based on cust number and lbu number
+        .assign(unique_ID = lambda x: x['CUSTOMER'] + x['ACCOUNT'])
 
+        # drop orginal lbu spend and credit
+        .drop(columns = f_cols + c_cols)
+        .rename(columns = new_cols)
+
+        )
 
+# exclude closed accounts
+financials = financial_without_credit[financial_without_credit['excldclosedacct'] == 'N']
+financials.columns = map(str.upper, financials.columns)
 #print(financial_without_credit.columns)
```

Comment:
You should remove commented out stuff instead of commenting it. This is what version control is for.